distribution/rules.bzl

```diff
@@ -0,0 +1,80 @@
+def _deploy_distribution_impl(ctx):
```

Comment:
@adammitchelldev can we have an Apache license header here as well?

Reply:
Will do